flax/experimental/nnx/nnx/graph_utils.py

```diff
@@ -147,6 +147,9 @@ class _HashableMapping(tp.Mapping[HA, HB], tp.Hashable):
   def __init__(self, mapping: tp.Mapping[HA, HB] | tp.Iterable[tuple[HA, HB]]):
     self._mapping = dict(mapping)
 
+  def __contains__(self, key: object) -> bool:
```

Comment:
Is this unrelated? I think `Mapping` implements `__contains__` in terms of other dunder methods.

Reply:
Yeah it does by calling getitem and catching a KeyError but that is super annoying for the debugger so I overwrote it.